jenkinsapi/build.py

```diff
@@ -160,10 +165,11 @@ def _get_git_rev(self):
         # Sometimes we have None as part of actions. Filter those actions
         # which have lastBuiltRevision in them
         _actions = [x for x in self._data['actions']
-                    if x and "lastBuiltRevision" in x]
+                    if x and "build" in x]
+        print(_actions)
```

Comment:
This fails build for Python 2.7. Please remove this line.

Reply:
@lechat Could you please help. I'm not sure why tests are failing.